### PR TITLE
add a readme for clickbench intro

### LINKED ISSUE #2902: Consider adding DataFusion to ClickBench benchmarks
**Is your feature request related to a problem or challenge? Please describe what you are trying to do.**
New benchmark suite! https://github.com/ClickHouse/ClickBench/

**Describe the solution you'd like**
Would be interesting to see how we compare

**Describe alternatives you've considered**
None

**Additional context**
None

### PR DESCRIPTION
**Pull request by waitingkuo**
# Which issue does this PR close?

Closes #2902 

 # Rationale for this change

initial work to add datafusion into <https://benchmark.clickhouse.com/>

# What changes are included in this PR?

# Are there any user-facing changes?

### CODE REVIEW

**Review by alamb — COMMENTED** [review]
Thanks @waitingkuo  -- rather than add something to the root directory, what would you think about putting this data into https://github.com/apache/arrow-datafusion/tree/master/benchmarks ?

I think it is similar in spirit to https://github.com/apache/arrow-datafusion/tree/master/benchmarks/db-benchmark

**Comment by waitingkuo** [comment]
@alamb  sure i can do it. let me spend sometime checking how db-benchmark works. should i move `conbench` as well?

**Comment by alamb** [comment]
@waitingkuo  moving conbench would be great

File: clickbench/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+2. cd ClickBench
+3. `bash run.sh`
+
+Note that this will output the needed result like the `result` field in <https://github.com/ClickHouse/ClickBench/blob/main/datafusion/results/f16s_v2.json>. We need to manually provide `system`, `date`, ... for now, and save a new json into `results` folder. After merged by ClickBench and new html files regenerated, it'll be shown in <https://benchmark.clickhouse.com/> .
+
+There're 43 queries to compute, each will be executed 3 times. each rows contains the execution time for these 3 quries. The overall output will be a 43 by 3 matrix.
+
+## To generate human readable results
+
+your can do
```
Comment (andygrove) [inline clickbench/README.md:39]:
```suggestion
you can do
```

File: clickbench/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+```bash
+bash run2.sh
+```
+
+Each query will be only exeute only once, for each query it'll print the SQL expression first then output the result. (Note that this index begins with 1, the ClickBench begins with 0)
```
Comment (andygrove) [inline clickbench/README.md:45]:
```suggestion
Each query will be executed only once, for each query it'll print the SQL expression first then output the result. (Note that this index begins with 1, the ClickBench begins with 0)
```

File: clickbench/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+<!---
```
Comment (andygrove) [inline clickbench/README.md:1]: I think that we should move this file to `benchmarks/clickbench/README.md` so that all the benchmarks are in one place.

**Review by andygrove — COMMENTED** [review]
Thanks for all of the work on this @waitingkuo. It is great to see results published.

**Comment by alamb** [comment]
FYI here is another approach to running clickbench in the datafusion repo: https://github.com/apache/arrow-datafusion/pull/7060

**Comment by alamb** [comment]
Closing as this PR is over a year old. Please feel free to reopen it / rebase it if you plan to keep working on it

**Pull request closed without merging**